### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by 0x41gilecat**
tests

### CODE REVIEW

File: .github/workflows/sh.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
 name: "Build Images"
 on: 
   workflow_call:
+  push:
```
Comment (0x41gilecat) [inline .github/workflows/sh.yml:4]: Review tests edit

**Review by 0x41gilecat — COMMENTED** [review]
Review

File: .github/workflows/sh.yml (hunk 1 above)
Comment (0x41gilecat) [reply .github/workflows/sh.yml:4]: Test

File: .github/workflows/sh.yml [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 
 jobs:
   init:
-    runs-on: test
+    runs-on: ubuntu-latest
```
Comment (hugo-syn) [inline .github/workflows/sh.yml:8]: Review 2

**Review by hugo-syn — CHANGES_REQUESTED** [review]
test

**Pull request closed without merging**